src/main/res/layout/loading_image_view.xml

```diff
@@ -7,7 +7,8 @@
         android:id="@+id/imageView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_gravity="center"/>
+        android:layout_gravity="center"
+        android:contentDescription="@string/loading_image_view_description"/>
```

Comment:
Would description of `Image` suffice, since the use case for this class is generic?